demo/api.html

```diff
@@ -9,16 +9,14 @@
       type="text/css"
       href="https://cdn.jsdelivr.net/npm/prismjs@latest/themes/prism.css"
     />
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```

Comment:
Please update this section to mirror this code example. 

https://github.com/AlaskaAirlines/auro-avatar/blob/main/demo/index.html#L12-L17